onnxruntime/core/providers/cuda/cuda_execution_provider.cc

```diff
@@ -142,6 +147,13 @@ CUDAExecutionProvider::CUDAExecutionProvider(const CUDAExecutionProviderInfo& in
     }
   }
 
+  // The user has requested that the default memory allocator be considered for shrinking at the end of every Run().
+  // This keeps the arena growth in check (as without this the arena will hang onto any growth permanently)
+  if (info_.default_memory_arena_cfg && info_.default_memory_arena_cfg->shrink_on_every_run) {
+    LOGS_DEFAULT(WARNING) << "The CUDA default memory allocator will be shrunk at the end of every Run";
```

Comment:
Is this something the user should be concerned about? Are we warning them or informing them as fyi? INFO?

Reply:
sure